samples/bluetooth/periodic_adv/src/main.c

```diff
@@ -56,7 +56,7 @@ void main(void)
 	}
 
 	while (true) {
-		k_sleep(K_SECONDS(1));
+		k_sleep(K_SECONDS(10));
```

Comment:
Would it make sense to sleep for the periodic advertising interval instead?

Reply:
Maybe, there was some weirdness though. because the periodic advertiser is using BT_LE_PER_ADV_DEFAULT which has BT_GAP_ADV_SLOW_INT_MIN/MAX. However the periodic advertising interval is specified in different units than the advertising interval (1.25 vs 0.625) so there is some confusion when the define documents it as 1.2 seconds, but it updates at 2.4 seconds.

Maybe best would be to have a parameter for the advertising interval and then just convert that to k_timeout value

Reply:
Actually, you don't know which one the controller will pick. Since you provide min/max intervals.

Reply:
Unless you of course provide the same min/max values. 

But you make a good point. 

> periodic advertiser is using BT_LE_PER_ADV_DEFAULT which has BT_GAP_ADV_SLOW_INT_MIN/MAX. However the periodic advertising interval is specified in different units than the advertising interval (1.25 vs 0.625) so there is some confusion when the define documents it as 1.2 seconds, but it updates at 2.4 seconds.

That sounds like something that needs to be fixed. Can you create an issue for it?

Reply:
Created this one: https://github.com/zephyrproject-rtos/zephyr/issues/37790

Feel free to update if you have additional input.